lib/msal-browser/src/cache/BrowserCacheManager.ts

```diff
@@ -490,7 +491,9 @@ export class BrowserCacheManager extends CacheManager {
             }
         });
 
+        debugger;
         this.internalStorage.clear();
+        await (this.cryptoImpl as CryptoOps).clearCache();
```

Comment:
This is another example of why I think the pop functions should be in their own class, instead of in `CryptoOps`. I think I'll make a PR demonstating what that might look like.